src/components/Search/SearchResults/SearchResults.tsx

```diff
@@ -16,6 +16,7 @@ export interface SearchResultsProps extends React.HTMLAttributes<HTMLElement> {
   loadingText?: string;
   onKeyDown?: (event: React.KeyboardEvent<HTMLAnchorElement>) => void;
   userInputValue?: string;
+  closeSearch: (value: boolean) => void;
```

Comment:
naming slightly confusing here, if it's accepting the boolean as a prop. could just keep it as `showSearch`
or, where you pass it in `Search.tsx`, do this:

```
closeSearch={() => showSearch(false)}
```